build-tests/heft-node-jest-test/tsconfig.json

```diff
@@ -0,0 +1,25 @@
+{
+  "$schema": "http://json.schemastore.org/tsconfig",
+
+  "compilerOptions": {
+    "outDir": "lib",
+    "rootDir": "src/",
+
+    "forceConsistentCasingInFileNames": true,
+    "jsx": "react",
+    "declaration": true,
+    "sourceMap": true,
+    "declarationMap": true,
+    "inlineSources": true,
+    "experimentalDecorators": true,
+    "strictNullChecks": true,
+    "noUnusedLocals": true,
+    "types": ["heft-jest", "node"],
```

Comment:
Note that `heft-jest` brings in the `jest` typings as well. With this design, projects do not need a `@types/jest` dependency, only `@types/heft-jest`.